src/routes/MapExplorer/MapExplorer.connector.test.js

```diff
@@ -0,0 +1,55 @@
+import orm from 'store/models'
+import { mapStateToProps } from './MapExplorer.connector'
+import { buildKey } from 'store/reducers/queryResults'
+import { FETCH_POSTS } from 'store/constants'
+import { times } from 'lodash/fp'
+
+describe('mapStateToProps', () => {
+  let state
+
+  beforeEach(() => {
+    const session = orm.session(orm.getEmptyState())
+
+    times(i => {
+      session.Post.create({ id: i.toString(), communities: ['1'] })
+    }, 5)
+
+    state = {
+      orm: session.state,
+      pending: {},
+      queryResults: {
+        [buildKey(FETCH_POSTS, { filter: 'foo' })]: {
+          ids: ['1', '3', '2'],
+          hasMore: true
+        }
+      }
+    }
+  })
+
+  it('returns empty posts if no results exist', () => {
+    expect(mapStateToProps(state, { postTypeFilter: 'bar' })).toHaveProperty('posts', [])
+  })
+
+  it('returns posts in the correct order', () => {
+    expect(mapStateToProps(state, { postTypeFilter: 'foo' })).toEqual(
+      expect.objectContaining({
+        hasMore: true,
+        pending: undefined,
+        posts: [
+          expect.objectContaining({ id: '1' }),
+          expect.objectContaining({ id: '3' }),
+          expect.objectContaining({ id: '2' })
+        ]
+      })
+    )
+  })
```

Comment:
This looks just fine, and am curious if you would like to move away from snapshot testing for these cases or more generally? (We don't need to discuss here, just noting these conversations for later as I notice them)

Reply:
no i think we should stick with them!